vulnerable_people_form/form_pages/nhs_registration_callback.py

```diff
@@ -14,11 +13,14 @@
     set_form_answers_from_nhs_user_info,
     load_answers_into_session_if_available
 )
+from ..integrations import google_analytics
+
+CONFIRMATION_URL = "/confirmation"
```

Comment:
as CONFIRMATION_URL isn't intended to be used outside this module it can be prefixed with underscore